<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigMultiEventState</class>
 <widget class="QWidget" name="ConfigMultiEventState">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
background:#35363C;
font-size:14px;
color:white;
}
QLineEdit{
border:none;
border-bottom:1px solid #5A636C;
padding-left:8px;
padding-right:8px;
}
QLineEdit:hover{
background:#4C4E56;
}
QLineEdit:focus{
border:1px solid #02BFFF;
}
QLabel[mstyle=&quot;hint&quot;]{
font-size:12px;color:#A0C2D3;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="line_config_box" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="1">
       <widget class="QLineEdit" name="input_signal_check">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>触发信号：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="input_trigger_func">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>信号参数检查：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>*设置信号参数检查函数，返回true时表示信号有效</string>
        </property>
        <property name="mstyle" stdset="0">
         <string>hint</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>*触发该分支的信号函数</string>
        </property>
        <property name="mstyle" stdset="0">
         <string>hint</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>*函数签名：
Q_INVOKABLE bool function(int signalIndex,const QVariantList&amp; data)</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="mstyle" stdset="0">
         <string>hint</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>384</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
